applications/serializers.py

```diff
@@ -155,6 +156,21 @@ class BootcampApplicationSerializer(serializers.ModelSerializer):
     bootcamp_run = BootcampRunSerializer()
     has_payments = serializers.SerializerMethodField()
     enrollment = serializers.SerializerMethodField()
+    certificate_link = serializers.SerializerMethodField()
+
+    def get_certificate_link(self, application):
+        """Returns certificate link if both certificate and certificate template are presentand enabled"""
+        if getattr(settings, "FEATURE_ENABLE_CERTIFICATES_FOR_USERS", False):
+            user = application.user
+            certificate = application.bootcamp_run.certificates.filter(
+                user=user
+            ).first()
```

Comment:
Normally first is used to obtain if they could be more than one object but in our case, one user can have one certificate.
It should be something like this
```
from django.core.exceptions import ObjectDoesNotExist
try:
     // return the link
except ObjectDoesNotExist:
     return None
```

Reply:
Don't you think `try-except` is an overhead in this case as we know that the filter will not return more than 1 record ever?

Reply:
no, it's not an overhead.

Reply:
Updated.